commands/create.go

```diff
@@ -110,7 +110,18 @@ func create(command *Command, args *Args) {
 	gh := github.NewClient(project.Host)
 
 	if gh.IsRepositoryExist(project) {
-		ui.Errorln("Existing repository detected. Updating git remote")
+
+		repo, err := gh.Repository(project)
```

Comment:
This queries `gh.Repository()` info again after `gh.IsRepositoryExist()` already queried it (but only returned a boolean). If you need to query whether a repository exist *and* access its info, it would be best to call `gh.Repository()` once initially and base all subsequent checks on that result.

Reply:
Good catch. Seeing that the `IsRepositoryExist` function is not used elsewhere, do you have objections to removing it?